index.html

```diff
@@ -180,7 +180,24 @@ <h2 class="connect">LET'S CONNECT</h2>
       </div>
       
     </div>
+
   </section>
 
+  <section class="contacts">
+    <h2 class="contact-me">Contact Me</h2> 
+    <p>If you have an application you <br> are interested in developing, a <br> feature that you need built or a <br> project that needs coding. I’d <br> love to help with it </p>
+    <form action="https://formspree.io/f/meqvlald" method="POST">
+      <label >
+        <input type="text" id="name" name="name" maxlength="30" placeholder="Name" required>
+      </label>
+      <label class="mail-input">
+        <input type="email" id="email" placeholder="Email" name="_replyto" required>
+      </label>
+      <label>
+      <textarea name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Write Your message here" required></textarea>
+      </label>
+      <button type="submit">Get in Touch</button>
```

Comment:
Submitting the form works just fine, but kindly add some interaction to the submit button. Buttons are multi-state objects and it's very important for users to get some visual feedback. The Figma template provides styling rules for `hover`, `active (when it's pressed)`, and `enabled (default state)`. Kindly implement this requirement.

Reply:
Thank you for your kind feedback.I have worked on the required changes, will work on the optional after program hours